### PR TITLE
OSS Friends list is out of date

### LINKED ISSUE #3181: OSS Friends list is out of date
## Bug Description

The OSS Friends list on Twenty.com is out of date.

For example, the official list includes Inbox Zero (and many others), but are not listed on Twenty:
https://twenty.com/oss-friends

The official list:
https://github.com/formbricks/formbricks/blob/ae530d710b3784b003a5b825b89646331cfdfbf4/apps/formbricks-com/pages/api/oss-friends/index.ts

### PR DESCRIPTION
**Pull request by gitstart-twenty**
Fixes #3181

### CODE REVIEW

**Comment by gitstart-twenty** [comment]
Hey @FelixMalfait,
We're using twenty-favicon for the icons here. However, some of the sites are still missing icons, it's possibly a bug in `twenty-favicon`. Should we open an issue for that? Or should we use something else for the icons?

Screenshot 2024-01-03 at 12 24 02

File: packages/twenty-website/src/app/components/oss-friends/Header.tsx [hunk 1]
```diff
@@ -0,0 +1,41 @@
+'use client';
```
Comment (FelixMalfait) [inline packages/twenty-website/src/app/components/oss-friends/Header.tsx:1]: Any idea why there's not coloring by Github on that file? There's probably something wrong but I can't figure out what?

Comment (FelixMalfait) [reply packages/twenty-website/src/app/components/oss-friends/Header.tsx:1]: The rest looks good to me. Haven't tried it locally yet. Does it behave well on mobile?

**Comment by gitstart-twenty** [comment]
> @gitstart-twenty Mmh indeed. Seems like there's an issue in the code. Feel free to assign that issue to yourself

Okay

File: packages/twenty-website/src/app/components/oss-friends/Header.tsx (hunk 1 above)
Comment (gitstart-twenty) [reply packages/twenty-website/src/app/components/oss-friends/Header.tsx:1]: Let me get you some screenshots @FelixMalfait

Comment (gitstart-twenty) [reply packages/twenty-website/src/app/components/oss-friends/Header.tsx:1]: Some mobile screenshots

localhost_3000_oss-friends(iPhone 12 Pro)

localhost_3000_oss-friends(Pixel 7) (2)

localhost_3000_oss-friends(Pixel 7)

Comment (gitstart-twenty) [reply packages/twenty-website/src/app/components/oss-friends/Header.tsx:1]: @FelixMalfait I've not figured out why the syntax highlighting is not working for this particular file. The  file has no visible errors

**Review by FelixMalfait — APPROVED** [review]
Great work thanks!

**Pull request merged**